metis-indexing/src/main/java/eu/europeana/indexing/IndexerImpl.java

```diff
@@ -142,18 +142,24 @@ public boolean indexTombstone(String rdfAbout, DepublicationReason depublication
     if (depublicationReason == DepublicationReason.LEGACY) {
       throw new IndexerRelatedIndexingException(
           format("Depublication reason %s, is not allowed", depublicationReason));
-    }
-    final FullBeanImpl publishedFullbean = this.connectionProvider.getIndexedRecordAccess().getFullbean(rdfAbout);
-    if (publishedFullbean != null) {
-      final FullBeanPublisher publisher = connectionProvider.getFullBeanPublisher(true);
-      final FullBeanImpl tombstoneFullbean = TombstoneUtil.prepareTombstoneFullbean(publishedFullbean, depublicationReason);
-      try {
-        publisher.publishTombstone(tombstoneFullbean, tombstoneFullbean.getTimestampCreated());
-      } catch (IndexingException e) {
-        throw new IndexerRelatedIndexingException("Could not create tombstone record '" + rdfAbout + "'.", e);
+    } else if (!(depublicationReason == DepublicationReason.GDPR
```

Comment:
Maybe simplify and use the values that are to be indexed? They are also 3.

Reply:
done. update to a positive `if`condition using the 3 values that are allowed.